### PR TITLE
Moar stylez

### PR DESCRIPTION
**Pull request by stephencarr**
New look and feel from marketing:

Screen Shot 2019-05-07 at 6 01 01 PM

- [x] The changes are not dumb

### CODE REVIEW

**Comment by alexinman** [comment]
is this responsive?

**Comment by stephencarr** [comment]
> is this responsive?

💯

**Comment by mike-stewart** [comment]
Much responsive. Very web 2.0.

File: index.html [hunk 1]
```diff
@@ -1,133 +1,175 @@
...
+          <td>
+            $10
+          </td>
+        </tr>
+        <tr>
+          <td>
+            Matthew Jones
+          </td>
+          <td>
+            $10
```
Comment (mike-stewart) [inline index.html:123]: @mdonjones 😱

Comment (JoshNorthrup) [reply index.html:123]: I think we need a test suite to avoid these unintended changes

**Pull request merged**